tests/unit/keyinput/test_basekeyparser.py

```diff
@@ -342,3 +344,13 @@ def test_respect_config_when_matching_counts(keyparser, config_stub):
 
     assert not keyparser._sequence
     assert not keyparser._count
+
+
+def test_count_limit_exceeded(handle_text, keyparser, caplog):
+    max_digits = sys.get_int_max_str_digits()
+    keys = (max_digits + 1) * (Qt.Key.Key_1,)
```

Comment:
nitpick: I'd prefer

```suggestion
    keys = (max_digits + 1) * [Qt.Key.Key_1]
```

because single-element tuples are a bit tricky syntax-wise for people new to Python.

Reply:
Good point. The single-item tuple syntax does look weird compared to a single-item list.